Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -139,6 +166,23 @@ class Navigator {
     }
 }
 
+extension Navigator: FallbackVersionsObserver {
+    
+    enum TileVersionState {
+        case nominal
```

Comment:
Would be good to have a comment describing what `nominal` `TileVersionState` means.